modules/appeals_api/spec/docs/v2/legacy_appeals_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+require Rails.root.join('spec', 'rswag_override.rb').to_s
+
+require 'rails_helper'
+require_relative '../../support/swagger_shared_components'
+
+# rubocop:disable RSpec/VariableName, RSpec/ScatteredSetup, RSpec/RepeatedExample, Layout/LineLength, RSpec/RepeatedDescription
+describe 'Legacy Appeals', swagger_doc: 'modules/appeals_api/app/swagger/appeals_api/v2/swagger.json', type: :request do
+  let(:apikey) { 'apikey' }
+
+  path '/legacy_appeals' do
+    get 'Returns eligible appeals in the legacy process for a Veteran.' do
+      tags 'Legacy Appeals'
+      operationId 'getLegacyAppeals'
+      security [{ apikey: [] }]
+      consumes 'application/json'
+      produces 'application/json'
+      description = 'Returns eligible Legacy Appeals for a Veteran. A Legacy Appeal is eligible if an SOC/SSOC ' \
+      'has been declared, and if the date of declaration is within the last 60 days. ' \
+      'SOC (Statement of Case) / SSOC (Supplemental Statement of Case)'
+      description description
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_ssn_header]
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_file_number_header]
```

Comment:
I think you'll want to update the description on their params...

Maybe something like this:
```
parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_ssn_header].merge({ required: false, description: 'Either X-VA-SSN or X-VA-File-Number is required' })
```
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/docs/v2/hlr_spec.rb#L212

Reply:
I see now!! Thank you!